app/views/admin/recent_events/_recent_events_form.html.slim

```diff
@@ -0,0 +1,32 @@
+= form_with model: [:admin , recent_event] do |f|
+  .card
+    .card-header
+      h5.card-title #{ recent_event.new_record? ? "New recent news post" : "Edit news post" }
+    .card-body
+      .form-group
+        = f.label :title, class: "form-label"
+        = f.text_field :title, class: "form-control #{ recent_event.title != nil ? recent_event.errors[:title].any? ? "is-invalid" : "is-valid" : "" }", "aria-describedby" => 'recentNewsTitle'
```

Comment:
But why, Ernest?? Why!!!? Kā to vispar var izlasit? Kāpēc "!= nil" ???

Es nezinu kāpēc tu uztaisiji tada veidā un pat negribu domat par to. Your time has come! 
Uztaisi jaunu form builderi un uztaisi viņam lauku text_input, kuram aprakstisi visu šo: invalid vai valid un errorus
https://api.rubyonrails.org/v6.1.0/classes/ActionView/Helpers/FormBuilder.html

Un turpmak izmantosi to. Vari viņu nosaukt kā BootstrapFormBuilder